styles/components/_filter-toggle.scss

```diff
@@ -0,0 +1,34 @@
+.cmp-filter-toggle {
+  margin-right: space("sm");
```

Comment:
Instead of setting a margin on each element, I think this a perfect use-case for using flexbox with the `gap` property.